ops/systemd/export_workers.env

```diff
@@ -1,16 +1,23 @@
 ## General settings
-HOSTNAME=https://export.hotosm.org
+HOSTNAME=export-staging.hotosm.org
 DJANGO_SETTINGS_MODULE=core.settings.project
 
 ## SMTP Settings
-EMAIL_HOST_PASSWORD=<SECURE_SMTP_PASSWORD>
 EMAIL_HOST=smtp.mailgun.org
-EMAIL_HOST_USER=export@examplemail.com
+EMAIL_HOST_USER=export@example.org
+EMAIL_HOST_PASSWORD=<password>
+REPLY_TO_EMAIL=sysadmin@example.org
+
+##OAUTH 
```

Comment:
Please take advantage of this comment to specify that it uses a OAuth1 app and not OAuth2